src/sass/_theme_rst.sass

```
@@ -239,22 +242,63 @@
   .hlist
     width: 100%
 
-  // Tables! Sphinx LOVES TABLES. Most of wyrm assumes you're only going to use a table as a table
-  // so I have to write a bunch of unique stuff for Sphinx to style them up differently like it's 2003.
-  table.docutils.citation, table.docutils.footnote
-    background: none
-    border: none
-    color: $text-medium
-    td, tr
+  // The html4 writer outputs the citation and footnotes as a table, and the
+  // html5 writer outputs these as a definition list. We will use the fairly
+  // well supported css `grid` attribute to make these back into a table
+  // HTML4 writer
+  html.writer-html4 &
+    table.docutils.citation, table.docutils.footnote
+      background: none
       border: none
-      background-color: transparent !important
-      white-space: normal
-    td.label
-      padding-left: 0
-      padding-right: 0
-      vertical-align: top
+      td, tr
+        border: none
+        background-color: transparent !important
+        white-space: normal
+      td.label
+        padding-left: 0
+        padding-right: 0
+        vertical-align: top
+  // HTML5 writer
+  html.writer-html5 &
+    dl dt span.classifier
+      &:before
+        content: " : "
+    dl.footnote,
+    dl.field-list
+      display: grid
+      grid-template-columns: max-content auto
```

```suggestion
      grid-template-columns: max-content auto
      > dt
        padding-left: 1rem
```

There is a small amount of left padding on the field lists in the HTML4 writer table output.

---

Oh yeah. I left this one because i think it looks better without the left padding. Any thoughts on which to use?

---

We can come back to this later, I'll just match the output.